src/bot/commands/general/disband.ts

```diff
@@ -0,0 +1,38 @@
+import { Command, CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { Message } from 'discord.js';
+import createPlayer from '../../../utils/create-player';
+import { isBound } from '../../../utils/isBound';
+import disbandEvent from '../../../actions/disband';
+
+export default class DisbandCommand extends Command {
+  public constructor(client: CommandoClient) {
+    super(client, {
+      name: 'disband',
+      memberName: 'disband',
+      group: 'general',
+      description: 'Disband your group (owner only)',
```

Comment:
Maybe add `guildOnly: true` here? Otherwise this will try to run in a DM and there will be no `msg.guild`